myhpi/static/scss/myHPI.scss

```diff
@@ -76,6 +76,16 @@ p {
     text-align: justify;
 }
 
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+#page {
+    flex: 1 0 auto;
+}

```

Comment:
Either use the bootstrap helper class or put it into `.page` (or remove `.page` and merge it into `#page`)